src/gui/SetupDialog.cpp

```diff
@@ -30,291 +31,247 @@
-	m_warnAfterSetup( !ConfigManager::inst()->value( "app",
-						"nomsgaftersetup" ).toInt() ),
-	m_displaydBFS( ConfigManager::inst()->value( "app", 
-		      				"displaydbfs" ).toInt() ),
-	m_MMPZ( !ConfigManager::inst()->value( "app", "nommpz" ).toInt() ),
-	m_disableBackup( !ConfigManager::inst()->value( "app",
-							"disablebackup" ).toInt() ),
-	m_openLastProject( ConfigManager::inst()->value( "app",
-							"openlastproject" ).toInt() ),
-	m_NaNHandler( ConfigManager::inst()->value( "app",
-							"nanhandler", "1" ).toInt() ),
-	m_hqAudioDev( ConfigManager::inst()->value( "mixer",
-							"hqaudio" ).toInt() ),
-	m_lang( ConfigManager::inst()->value( "app",
-							"language" ) ),
-	m_workingDir( QDir::toNativeSeparators( ConfigManager::inst()->workingDir() ) ),
-	m_vstDir( QDir::toNativeSeparators( ConfigManager::inst()->vstDir() ) ),
-	m_artworkDir( QDir::toNativeSeparators( ConfigManager::inst()->artworkDir() ) ),
-	m_ladDir( QDir::toNativeSeparators( ConfigManager::inst()->ladspaDir() ) ),
-	m_gigDir( QDir::toNativeSeparators( ConfigManager::inst()->gigDir() ) ),
-	m_sf2Dir( QDir::toNativeSeparators( ConfigManager::inst()->sf2Dir() ) ),
+SetupDialog::SetupDialog(ConfigTabs tab_to_open) :
+	m_displaydBFS(ConfigManager::inst()->value(
+			"app", "displaydbfs").toInt()),
+	m_tooltips(!ConfigManager::inst()->value(
+			"tooltips", "disabled").toInt()),
+	m_displayWaveform(ConfigManager::inst()->value(
+			"ui", "displaywaveform").toInt()),
+	m_printNoteLabels(ConfigManager::inst()->value(
+			"ui", "printnotelabels").toInt()),
+	m_compactTrackButtons(ConfigManager::inst()->value(
+			"ui", "compacttrackbuttons").toInt()),
+	m_oneInstrumentTrackWindow(ConfigManager::inst()->value(
+			"ui", "oneinstrumenttrackwindow").toInt()),
+	m_MMPZ(!ConfigManager::inst()->value(
+			"app", "nommpz").toInt()),
+	m_disableBackup(!ConfigManager::inst()->value(
+			"app", "disablebackup").toInt()),
+	m_openLastProject(ConfigManager::inst()->value(
+			"app", "openlastproject").toInt()),
+	m_lang(ConfigManager::inst()->value(
+			"app", "language")),
+	m_saveInterval(	ConfigManager::inst()->value(
+			"ui", "saveinterval").toInt() < 1 ?
+			MainWindow::DEFAULT_SAVE_INTERVAL_MINUTES :
+			ConfigManager::inst()->value(
+			"ui", "saveinterval").toInt()),
+	m_enableAutoSave(ConfigManager::inst()->value(
+			"ui", "enableautosave", "1").toInt()),
+	m_enableRunningAutoSave(ConfigManager::inst()->value(
+			"ui", "enablerunningautosave", "0").toInt()),
+	m_smoothScroll(ConfigManager::inst()->value(
+			"ui", "smoothscroll").toInt()),
+	m_animateAFP(ConfigManager::inst()->value(
+			"ui", "animateafp", "1").toInt()),
+	m_vstEmbedMethod(ConfigManager::inst()->vstEmbedMethod()),
+	m_vstAlwaysOnTop(ConfigManager::inst()->value(
+			"ui", "vstalwaysontop").toInt()),
+	m_syncVSTPlugins(ConfigManager::inst()->value(
+			"ui", "syncvstplugins").toInt()),
+	m_disableAutoQuit(ConfigManager::inst()->value(
+			"ui", "disableautoquit").toInt()),
```

Comment:
I just noticed the default value of `m_syncVSTPlugins` and `m_disableAutoQuit` was 1 before this PR, but it's 0 now. This PR should have not been touched the default value.
Sorry for not noticing this before.

Reply:
Oh, sorry about that.
I should just add `"1"` as the last argument to `value()`, right?

Reply:
Yes. That's what you only need to do.

Reply:
Addressed in https://github.com/LMMS/lmms/commit/9f0baab93ae18f88c07dff76a0563f73200b649f.